apps/src/applab/applab.js

```diff
@@ -962,6 +975,16 @@ Applab.isRunning = function () {
   return $('#resetButton').is(':visible') || studioApp.share;
 };
 
+Applab.toggleDivApplab = function(isVisible) {
```

Comment:
Could use a short function description (even though this one is pretty simple). Code likes this makes me wish we had more stuff reactified as it makes this sort of stuff so easy :)